grammars/ini.cson

```diff
@@ -95,5 +99,9 @@
         'name': 'punctuation.definition.string.end.ini'
     'name': 'string.quoted.double.ini'
   }
+  {
```

Comment:
What is the purpose of this? It seems to catch everything and will just highlight everything as this definition.

Reply:
This will catch all strings that are after the "=" sign and not contained in quotes ("", ''). 
And the reason I moved it in the end is to avoid exactly what you suggest. So, because it is in the end, it will catch everything after the "=" that hasn't been caught by all the previous expressions. 
Have you tried it to see how it behaves?

Reply:
> This will catch all strings that are after the "=" sign and not contained in quotes ("", '').

I don't think that's all it will do as it also matches strings like `foo = bar` and `  baz == fizz` without any anchors or delimiters.

> So, because it is in the end, it will catch everything after the "=" that hasn't been caught by all the previous expressions.

I don't know whether this is desirable. On top of what you intend it to do, it also has the side effect of covering up any expressions that have failed to match but should have.

Unless we have a pressing reason to do this, I'm thinking we remove it.

Reply:
You are right about this. No pressure. Let's remove it for now and I will try to improve it and come back at a later stage. Thanks